README.md

```diff
@@ -45,40 +44,40 @@ Add the dependency
    }
 ```
 
-И зависимость на библиотеку
+And the dependency on the library
 
 ```
    dependencies {
-    	implementation 'com.github.codex-team:hawk.kotlin:v1.0.4'
+    	implementation 'com.github.codex-team:hawk.kotlin:v3.0'
    }
 ```
 
-### Пример использования
+### How to use
 
-Для активации прослушки ошибок, вы можете добавить в ваш главный класс следующий код
+To activate the error collector, you can add the following code to your main class
 
 ```kotlin
-val catcher = HawkCatcher(integrationToken)
+val catcher: HawkExceptionCatcher = HawkCatcher(integrationToken)
     .versionProvider(VersionProviderImpl())
     .userProvider(UserProviderImpl())
```

Comment:
Add an explanation of these providers? What are they're allowing to do? How they should be implemented? Where we can see examples?